### PR TITLE
Api endpoint for user webhooks

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
Added an API endpoint so that users would be able to access their webhook endpoints list.

## Related Tickets & Documents
#3715

### CODE REVIEW

**Comment by lightalloy** [comment]
@ben @rhymes Agree, I'll add tests.

File: app/views/api/v0/webhooks/index.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,8 @@
+json.array! @webhooks.each do |endpoint|
+  json.id             endpoint.id
```
Comment (rhymes) [inline app/views/api/v0/webhooks/index.json.jbuilder]: let's add the `json.type_of            "webhook"` field just for consistency with the other APIs

**Comment by lightalloy** [comment]
@rhymes why are we putting `type_of` (not `type` e.g.) in the json ?

**Comment by rhymes** [comment]
@lightalloy honestly? For consistency with the endpoints that already existed. Personally I'm not a fan 😂

**Comment by lightalloy** [comment]
@rhymes oh, I see. I noticed that not all of the endpoints include `type_of`, so I supposed it was your idea.

**Pull request merged**